common/src/test/scala/explore/common/arb/ArbTargetObsQueries.scala

```diff
@@ -11,7 +11,7 @@
 // import lucuma.core.enums.ObsActiveStatus
 // import lucuma.core.enums.ObsStatus
 // import lucuma.core.model.Asterism
-// import lucuma.core.model.Observation
+// import explore.model.Observation
```

Comment:
🤔

Reply:
`lucuma.core.model.Observation` only has the `Id` type, which is now forwarded from `explore.model.Observation`.

We use this same technique in other cases (eg: `Visit`).